tests/test_lint_makefile.py

```
@@ -3,17 +3,22 @@
 from make_profiler import parser, lint_makefile
 
 
-def test_missing_rule(capsys):
-    mk = "all: foo\n"
+def run_validation(mk: str) -> tuple[bool, list[lint_makefile.LintError]]:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)